src/System.Net.Security/src/System/Net/SecureProtocols/SslStream.cs

```diff
@@ -139,20 +126,6 @@ internal virtual void EndAuthenticateAsClient(IAsyncResult asyncResult)
         //
         // Server side auth.
         //
-        public virtual void AuthenticateAsServer(X509Certificate serverCertificate)
-        {
-            AuthenticateAsServer(serverCertificate, false, SecurityProtocol.DefaultSecurityProtocols, false);
-        }
-
-        public virtual void AuthenticateAsServer(X509Certificate serverCertificate, bool clientCertificateRequired,
-                                               SslProtocols enabledSslProtocols, bool checkCertificateRevocation)
-        {
-            SecurityProtocol.ThrowOnNotAllowed(enabledSslProtocols);
-
-            _sslState.ValidateCreateContext(true, string.Empty, enabledSslProtocols, serverCertificate, null, clientCertificateRequired, checkCertificateRevocation);
-            _sslState.ProcessAuthentication(null);
```

Comment:
No, we can't remove methods like `ValidateCreateContext` and `ProcessAuthentication` because they are in the internal APM async implementation.